examples/create_identity_provider/gitlab/README.md

```diff
@@ -0,0 +1,37 @@
+# GitLab identity provider example
+
+Configuring GitLab authentication allows users to log in to OpenShift Container Platform with their GitLab credentials.
+
+If you use GitLab version 7.7.0 to 11.0, you connect using the [OAuth integration](http://doc.gitlab.com/ce/integration/oauth_provider.html). If you use GitLab version 11.1 or later, you can use [OpenID Connect](https://docs.gitlab.com/ce/integration/openid_connect_provider.html) (OIDC) to connect instead of OAuth.
+This example shows how to create a Google identity provider.
+
+## Prerequisites
+
+### An OpenShift cluster
+
+This example assumes you have created a cluster via OCM, It can be via this terraform provider or by a different client.
+We will need the cluster name and some credentials.
+
+### Setting up your application in GitLab
+
+You will need a client ID/Secret of a [registered GitLab OAuth application](https://docs.gitlab.com/ce/api/oauth2.html). The application must be configured with a callback URL of `https://oauth-openshift.apps.<cluster-name>.<cluster-domain>/oauth2callback/<idp-provider-name>`
```

Comment:
In Github you added here the actual name and here you added `<idp-provider-name>`
Please be consistent

Reply:
Ill add the example here as well